backend/app/assets/stylesheets/spree/backend/plugins/_select2.scss

```diff
@@ -90,3 +90,12 @@
     * { color: $dropdown-link-hover-color }
   }
 }
+
+.select2-choices::after {
```

Comment:
It also looks off on pages where the input expands to several lines.

![](http://i.hawth.ca/s/kl8dwRhw.png)